### ISSUE TITLE
adapt-youtube: example.json prevents page from loading

### ISSUE THREAD

**Issue by chucklorenz** [cgkineo/adapt-youtube#18]
FW v5.17.1
YouTube v2.1.0

I copied the example.json into place in components.json and only updated the ids. I get no compile error when building. But the page doesn't load. No errors reported in the dev tools console.

To be clear, this is the page that is not loading; it's not an issue of the video not loading.

Since the README provides little practical information, I tried replacing _media._source with a different video's url--following the same model. Again, the page did not load.

**Comment by oliverfoster** [comment]
Can you try / review this pr please? @chucklorenz 

https://github.com/adaptlearning/adapt-youtube/pull/17

**Comment by chucklorenz** [comment]
@oliverfoster 
Tried, but still won't load.
Also attempted 
From my console:
![image](https://user-images.githubusercontent.com/9489014/137729485-51cacb33-f0e2-4300-9539-c557ccdaec53.png)
And from the cross-site cookie error:
![image](https://user-images.githubusercontent.com/9489014/137730129-3821041c-7fff-4700-a789-3a49630505cb.png)

Happy to test/try other suggestions, just let me know...

**Comment by oliverfoster** [comment]
The cookie bits are just warnings, they can be ignored.

References:
https://stackoverflow.com/questions/62984682/cannot-embed-youtube-video-cookie-error-samesite-attribute
https://cookie-script.com/blog/how-to-add-youtube-videos-without-cookies
https://axbom.com/embed-youtube-videos-without-cookies/

Works for me:
![youtube1](https://user-images.githubusercontent.com/7974663/137876279-db1a0cb6-2e7a-4bbe-a4f7-3c381f85fc98.gif)

Example build:
[18-youtube-example.zip](https://github.com/adaptlearning/adapt-youtube/files/7371698/18-youtube-example.zip)

**Comment by chucklorenz** [comment]
Ok, I’ll go through the course today to see what’s causing interference/conflict.

**Comment by chucklorenz** [comment]
I believe googleAnalytics was involved.

- I noticed a [warning in YouTubeView.js](https://github.com/adaptlearning/adapt-youtube/blob/c48853d82f42a8c1951c04b2afe82201b382c5cb/js/YouTubeView.js#L102-L107). I logged out of YouTube, but the page still failed to load.
- I remembered I had googleAnalytics installed. I added it to the build excludes, and the page loaded with the YouTube component working. 
- I returned to config.json to see if toggling "_isEnabled" on googleAnalytics would have a consistent impact. It does not; adapt-youTube (and the page) continues to load as expected. Both youTube and googleAnalytics appear to be function correctly now.  So I'm guessing that having googleAnalytics installed first somehow interfered with the proper loading/compile order. But admittedly, I'm out of my depth.

Thanks, @oliverfoster
